src/sagemaker_xgboost_container/algorithm_mode/serve.py

```diff
@@ -57,11 +70,8 @@ def load(self):
 
 class ScoringService(object):
     PORT = os.getenv("SAGEMAKER_BIND_TO_PORT", 8080)
-    # NOTE: 6 MB max content length
-    MAX_CONTENT_LENGTH_MB = int(os.getenv("MAX_CONTENT_LENGTH", '6'))
-    MAX_CONTENT_LENGTH = MAX_CONTENT_LENGTH_MB * 1024 ** 2
-
     MODEL_PATH = os.getenv(sm_env_constants.SM_MODEL_DIR)
+    MAX_CONTENT_LENGTH = PARSED_MAX_CONTENT_LENGTH
```

Comment:
Is there any reason to have both `MAX_CONTENT_LENGTH` and `PARSED_MAX_CONTENT_LENGTH`? It looks like the only time you use `PARSED_MAX_CONTENT_LENGTH` is in line 179, when setting the value of `MaxPayloadInMB`. I would get rid of `PARSED_MAX_CONTENT_LENGTH` and set `MAX_CONTENT_LENGTH` directly to the value of `_get_max_content_size()`.

Reply:
Settled offline, ScoringService has MAX_CONTENT_LENGTH and also would like to avoid parsing env vars more than once.